diff --git a/Makefile b/Makefile
index 0256010..7dcbd90 100644
--- a/Makefile
+++ b/Makefile
@@ -4,7 +4,7 @@ VERSION := $(or $(VERSION), $(shell git describe --dirty --always --tags 2>/dev/

 PKG_CONFIG ?= pkg-config
 CFLAGS += -O2 -Wall -g `$(PKG_CONFIG) --cflags libxml-2.0 libusb-1.0`
-LDFLAGS += `$(PKG_CONFIG) --libs libxml-2.0 libusb-1.0`
+LDFLAGS += `$(PKG_CONFIG) --libs libxml-2.0 libusb-1.0 --static`
 ifeq ($(OS),Windows_NT)
 LDFLAGS += -lws2_32
 endif
