diff --git a/Makefile b/Makefile
index ddd06c9..bef69c4 100644
--- a/Makefile
+++ b/Makefile
@@ -3,7 +3,7 @@ RAMDUMP := qdl-ramdump
 VERSION := $(or $(VERSION), $(shell git describe --dirty --always --tags 2>/dev/null), "unknown-version")
 
 CFLAGS += -O2 -Wall -g `pkg-config --cflags libxml-2.0 libusb-1.0`
-LDFLAGS += `pkg-config --libs libxml-2.0 libusb-1.0`
+LDFLAGS += `pkg-config --libs libxml-2.0 libusb-1.0 --static`
 ifeq ($(OS),Windows_NT)
 LDFLAGS += -lws2_32
 endif
